lib/hydra/derivatives.rb

```diff
@@ -118,7 +118,7 @@ def constantize_processor(processor)
     end
 
     def transform_datastream(file_name, transform_directives, opts={})
```

Comment:
that was my initial change and then I created and rspec that called
transform_file to make sure it worked, and from what I can tell it does.
Was that overkill?

On Mon, Aug 10, 2015 at 12:09 PM, Colin <notifications@github.com> wrote:

> In lib/hydra/derivatives.rb
> <https://github.com/projecthydra/hydra-derivatives/pull/69#discussion_r36649320>
> :
>
> > @@ -118,7 +118,7 @@ def constantize_processor(processor)
> >      end
> >
> >      def transform_datastream(file_name, transform_directives, opts={})
>
> I think empty opts are still valid. The issue was that the transform_file
> call from within transform_datastream was not actually passing the opts
> along. It was initializing a new empty hash no matter what you passed in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra/hydra-derivatives/pull/69/files#r36649320>
> .
>